pages/countries/index.js

```diff
@@ -20,24 +25,55 @@ export default function Countries({ surfspots }) {
     router.push(`/`);
   }
 
+  function handleChange(event) {
+    setSearchTerm(event.target.value);
+    let searchValue = event.target.value;
+    setSearchedSpots(
+      surfspots.filter((spot) => spot.slug.includes(searchValue.toLowerCase()))
+    );
+
+    if (event.target.value === "") {
+      setSearchTerm(false);
+    } else {
+      setSearchTerm(true);
+    }
+  }
+
   return (
     <>
       <Head>
         <title>SpotiFinder</title>
       </Head>
-      <Header onMapShown={handleMapShown} usedMapShown={false} />
+      <MenueHeader onMapShown={handleMapShown} usedMapShown={false} />
       <ListSurfspots>
-        <FavoriteSpotsListItem name={"Favoriten"} link={`/favoritespots`} />
-        {uniqueCountrys.map((uniqueCountry) => {
-          const countryName = capitalizeFirstLetter(uniqueCountry);
-          return (
-            <ListSurfspotsItems
-              link={`/countries/${uniqueCountry}`}
-              name={countryName}
-              key={uniqueCountry}
-            />
-          );
-        })}
+        <SearchSpot onChange={handleChange} />
+        {searchTerm ? (
+          <>
```

Comment:
brauchst keine Fragments hier

Reply:
Doch leider schon, dadurch dass ich in der abfrage wieder eine abfrage habe.. :)